### PR TITLE
 B-03762 Elemental Chat handles Web-SDK Signals

### PR DESCRIPTION
**Pull request by timotree3**

### CODE REVIEW

**Comment by timotree3** [comment]
TODO: Add integration test

**Comment by timotree3** [comment]
Turns out test is not needed

**Comment by timotree3** [comment]
Depends on signals in WebSDK Holo-Host/web-sdk#20

**Comment by robbiecarlton** [comment]
> Depends on signals in WebSDK [Holo-Host/web-sdk#20](https://github.com/Holo-Host/web-sdk/pull/20)

@timotree3 - 0.3.0 of WebSDK  is now released

File: package.json [hunk 1]
```diff
@@ -9,12 +9,10 @@
     "serve": "VUE_APP_UI_VERSION=$npm_package_version VUE_APP_CONTEXT='holo-host' VUE_APP_CHAPERONE_SERVER_URL='http://localhost:24273/' vue-cli-service serve",
     "build": "VUE_APP_UI_VERSION=$npm_package_version VUE_APP_CONTEXT='holo-host' VUE_APP_CHAPERONE_SERVER_URL='https://chaperone.holo.host/' vue-cli-service build",
     "build-self-hosted": "VUE_APP_UI_VERSION=$npm_package_version VUE_APP_CONTEXT='self-hosted' vue-cli-service build",
-    "test:unit": "vue-cli-service test:unit",
-    "test:e2e": "vue-cli-service test:e2e",
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "@holo-host/web-sdk": "^0.2.2",
+    "@holo-host/web-sdk": "git://github.com/Holo-Host/web-sdk.git#67ab8646fecb15835d535f496081c2b3b3e43795",
```
Comment (timotree3) [inline package.json:15]:
```suggestion
    "@holo-host/web-sdk": "^0.3.0",
```

**Comment by timotree3** [comment]
Getting this error on commit:
```
> running pre-commit hook: lint-staged
/home/timo/dev/hc/elemental-chat-ui/node_modules/any-observable/register.js:29
		throw new Error('Cannot find any-observable implementation nor' +
		^

Error: Cannot find any-observable implementation nor global.Observable. You must install polyfill or call require("any-observable/register") with your preferred implementation, e.g. require("any-observable/register")('rxjs') on application load prior to any require("any-observable").
    at loadImplementation (/home/timo/dev/hc/elemental-chat-ui/node_modules/any-observable/register.js:29:9)
    at /home/timo/dev/hc/elemental-chat-ui/node_modules/any-observable/loader.js:30:18
    at Object.<anonymous> (/home/timo/dev/hc/elemental-chat-ui/node_modules/any-observable/index.js:2:39)
    at Module._compile (node:internal/modules/cjs/loader:1083:30)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1112:10)
    at Module.load (node:internal/modules/cjs/loader:948:32)
    at Function.Module._load (node:internal/modules/cjs/loader:789:14)
    at Module.require (node:internal/modules/cjs/loader:972:19)
    at require (node:internal/modules/cjs/helpers:88:18)
    at Object.<anonymous> (/home/timo/dev/hc/elemental-chat-ui/node_modules/@samverschueren/stream-to-observable/index.js:2:20)
```

**Comment by timotree3** [comment]
Not anymore after rebasing...

File: package.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "@holo-host/web-sdk": "^0.2.2",
+    "@holo-host/web-sdk": "~0.3.0",
```
Comment (timotree3) [inline package.json:17]:
```suggestion
    "@holo-host/web-sdk": "~0.3.1",
```

**Review by robbiecarlton — APPROVED** [review]
LGTM

**Pull request merged**